src/Tizen.NUI/src/public/Common/BaseHandle.cs

```diff
@@ -680,7 +680,8 @@ private void disposeDebuggingDispose(DisposeTypes type)
 
             if (swigCPtr.Handle != IntPtr.Zero && swigCPtrCopy.Handle != IntPtr.Zero && HasBody())
             {
-                var process = global::System.Diagnostics.Process.GetCurrentProcess().Id;
+                using var currentProcess = global::System.Diagnostics.Process.GetCurrentProcess();
```

Comment:
Is this right usage of using..?
I've only seen the below code usages.
using (var currentProcess = global::System.Diagnostics.Process.GetCurrentProcess())
{
             ....
}

Reply:
and one more thing. If we use `using` tag, `currentProcess` will be Automatically disposed. I don't know exactly but, Is it intention?

Reply:
I think this svace is about CA1837.
https://docs.microsoft.com/ko-kr/dotnet/fundamentals/code-analysis/quality-rules/ca1837

If so, Environment.ProcessId is better than Process.GetCurrentProcess().Id as follows.
```
var process = global::System.Environment.ProcessId;
```

Reply:
> I think this svace is about CA1837. https://docs.microsoft.com/ko-kr/dotnet/fundamentals/code-analysis/quality-rules/ca1837
> 
> If so, Environment.ProcessId is better than Process.GetCurrentProcess().Id as follows.
> 
> ```
> var process = global::System.Environment.ProcessId;
> ```

Thank you for review, but I got build error. 
src/public/Common/BaseHandle.cs(683,58): error CS0117: 'Environment' does not contain a definition for 'ProcessId' [/home/ssong2best/dDrive/github.com/dongsug-song/tizenfx/develNUI/TizenFX/src/Tizen.NUI/Tizen.NUI.csproj]

Reply:
> Is this right usage of using..? I've only seen the below code usages. using (var currentProcess = global::System.Diagnostics.Process.GetCurrentProcess()) { .... }

Thank you for review, I can see this =>
in https://docs.microsoft.com/en-us/dotnet/csharp/language-reference/keywords/using-statement
**"The using declaration, which was introduced in C# 8.0, doesn't require braces:"**
and only process id was needed, so I think it would be OK after process id is gotten, it can be disposed.

Reply:
> and one more thing. If we use `using` tag, `currentProcess` will be Automatically disposed. I don't know exactly but, Is it intention?


Thank you for review, I can see this =>
in https://docs.microsoft.com/en-us/dotnet/csharp/language-reference/keywords/using-statement
"The using declaration, which was introduced in C# 8.0, doesn't require braces:"
**and only process id was needed, so I think it would be OK after process id is gotten, it can be disposed.
only process id was used.**

Reply:
> > and one more thing. If we use `using` tag, `currentProcess` will be Automatically disposed. I don't know exactly but, Is it intention?
> 
> Thank you for review, I can see this => in https://docs.microsoft.com/en-us/dotnet/csharp/language-reference/keywords/using-statement "The using declaration, which was introduced in C# 8.0, doesn't require braces:" **and only process id was needed, so I think it would be OK after process id is gotten, it can be disposed. only process id was used.**

good to know!! thank you for sharing!